README.md

```diff
@@ -77,6 +77,7 @@ There are command options available:
 
 | option         | input format | default | description |
 |----------------|--------------|---------|-------------|
+| `--audio_preprocess_type`   | {MFCCs, PCEN}       | PCEN     | type of audio preprocess to use            |
```

Comment:
If the new default is PCEN, then we'll need to update the existing instructions to use MFCCs.